pkg/accessor/events/events_accessor.go

```diff
@@ -0,0 +1,93 @@
+package events
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/eventbridge"
+	"github.com/aws/aws-sdk-go/service/eventbridge/eventbridgeiface"
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+)
+
+type eventsAccessor struct {
+	client eventbridgeiface.EventBridgeAPI
+}
+
+// https://docs.aws.amazon.com/sdk-for-go/api/service/eventbridge/#EventBridge.PutRule
+func (this *eventsAccessor) CreateEventRule(name, eventBusName, eventPattern *string) (*string, error) {
+	putEventRuleInput := &eventbridge.PutRuleInput{
+		Name:         name,
+		EventBusName: eventBusName,
+		EventPattern: eventPattern,
+	}
+
+	response, err := this.client.PutRule(putEventRuleInput)
+
+	if err != nil {
+		return nil, err
+	}
+
+	ruleArn := response.RuleArn
+	return ruleArn, nil
+}
+
+func (this *eventsAccessor) PutEvents(eventBusName, detail, detailType, source, resourceName *string) (*eventbridge.PutEventsOutput, error) {
+
+	resources := make([]*string, 1)
+	resources[0] = resourceName
+	putEventRequestEntryInput := &eventbridge.PutEventsRequestEntry{
+		Detail:       detail,
+		DetailType:   detailType,
+		EventBusName: eventBusName,
+		Resources:    resources,
+		Source:       source,
+	}
+
+	//var putEventRequestEntryInputArr []*eventbridge.PutEventsRequestEntry
+	putEventRequestEntryInputArr := make([]*eventbridge.PutEventsRequestEntry, 1)
+	putEventRequestEntryInputArr[0] = putEventRequestEntryInput
+
+	putEventsInputRequest := &eventbridge.PutEventsInput{
+		Entries: putEventRequestEntryInputArr,
+	}
+
+	result, err := this.client.PutEvents(putEventsInputRequest)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return result, err
```

Comment:
Consider inlining to simplify.
```suggestion
	return this.client.PutEvents(&eventbridge.PutEventsInput{
	   ...
	})
```